components/footer.tsx

```
@@ -1,29 +1,58 @@
-            className="w-[100px]"
-            src={
-              theme === "light"
-                ? "/images/logo-netlify-dark.png"
-                : "/images/logo-netlify-light.png"
-            }
-            alt="Netlify"
+      <div className="max-w-screen-lg flex items-center gap-8 mx-auto">
+        <div className="max-w-[480px]">
+          <p
+            className="text-sm text-muted-foreground"
+            dangerouslySetInnerHTML={{ __html: content.citation.text }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)